foster55f/market2table-fe
# Testing 1

### What does this PR do?
 - This PR adds snapshot testing to components
 - updates some useSelectors to deconstruct from state
 - all tests are passing

### How to test?
 - pull down (clone down) and run npm install and npm start
 - run `npm test` too see tests that are passing

### Issues:
 - Still need to test more to get percentage up.
